test/system/030-run.bats

```diff
@@ -31,4 +31,12 @@ echo $rand        |   0 | $rand
     done < <(parse_table "$tests")
 }
 
+@test "podman run - uidmapping has no /sys/kernel mounts" {
+      run_podman $expected_rc run --uidmapping 0:100:10000 $IMAGE mount | grep /sys/kernel
```

Comment:
I think you need --net=host here to make the current behaviour happen.

Reply:
I've added another test with --net=host, so that we can test both cases